### ISSUE TITLE
On mobiles

### ISSUE THREAD

**Issue by Albancee** [kombai/freewall#206]
Hello, i need blocks with 300 width, but on mobile version blocks vanish like in this example http://vnjs.net/www/project/freewall/example/demo-filter.html

I think this is bug, please help me to solve this.... Btw here is the code:
```
    jQuery(function() {
        jQuery(".free-wall").each(function() {
            var wall = new freewall(this);;
            wall.reset({
                selector: '.size320',
                animate: true,
                cellW: 200,
                cellH: 'auto',
                onResize: function() {
                    wall.fitWidth();
                }
            });
            wall.fitWidth();
        });
    });
```

**Comment by kombai** [comment]
Yes, it hide because the width of block  less than one unit width (200px + padding + margin..~=300px in your example).
You can easy fix that by using cellW as a function:
```js
 cellW: function(container) {
  var containerW = container.with();
  if (containerW > 300) {
  return 200; 
 } else {
 return 2; 
}
}

```  

Please take a look on this example:
http://vnjs.net/www/project/freewall/example/live-size.html
Best

**Comment by schizflux** [comment]
Hello Kombai,

I have the same problem: My blocks size33 and size36 disappear on screens  320x480 (iphone6). Top Left image, and the block of text on the right as seen on this page:

http://scorpios.schizflux.org/secondary.htm

Unfortunately my knowledge of programming is very limited.  Will you please point me to the file and the line of code that I need to change to solve it and how? Thank you!
